pkgs/mime/lib/src/mime_type.dart

```diff
@@ -53,7 +41,7 @@ class MimeTypeResolver {
   /// though a file have been saved using the wrong file-name extension. If less
   /// than [magicNumbersMaxLength] bytes was provided, some magic-numbers won't
   /// be matched against.
-  String? lookup(String path, {List<int>? headerBytes}) {
+  String? lookupMimeType(String path, {List<int>? headerBytes}) {
```

Comment:
If we are adding new members, is now a good time to change the name to `lookupMediaType`?